far/changelog

```diff
@@ -1,3 +1,12 @@
+--------------------------------------------------------------------------------
+MZK 2023-06-20 19:53:29-07:00 - build 6167
+
+1. More ways to scroll horizontally Find All list items in the internal Editor.
```

Comment:
> Find All list items in the internal Editor

Why so precise?

Reply:
If you mean why it is so narrowly limited to Find All list only, it's because most all other lists do not need this feature. I wanted to fix Find All. Actual implementation is in `VMenu`, so I added new keys there, but I still think of the feature as of Find-All-specific. Historical, you know.

Updated everything. Thanks.

Reply:
I mean the implementation is generic and works everywhere, so saying that "it's find all specifically" can be confusing and just "menu items" (or "menu/listbox items" even) should cause less questions.

Thank you.